certora/specs/sharesAssetsConversion.spec

```diff
@@ -0,0 +1,257 @@
+import "erc20.spec"
```

Comment:
some of these are ERC4626 properties from the EIP.
We would like to move them to the ERC4626 spec rather than gadi's spec